docs/dl-getting-started.md

```diff
@@ -114,6 +114,13 @@ In the contract argument, the value specified with the key `asset_id` must be un
 In Scalar DL 1.0, `asset_id` is not a reserved json key name and you can use any json key name, but this will probably be changed in future versions.
 (`asset_id` and `asset_ids` will be reserved for future enhancements.)
 
+## Create your contracts
```

Comment:
There is already a "Create a contract" section above. I think the point of this section is the link to the emulator so perhaps it should be renamed to "Scalar DL Emulator" and expanded from there.

Reply:
My intention is not only for Emulator. 
I feel we also want to say "create your contract" instead of creating what's given.
Let me revise the sentence and the section title.

Reply:
Ah, I see. Maybe then the section title could be "Create your own contracts" or something?